test/Driver/batch_mode_remark.swift

```diff
@@ -0,0 +1,12 @@
+// Ensure that driver issues a remark iff in batch mode.
+//
+// RUN: %swiftc_driver -whole-module-optimization -enable-batch-mode   %S/../Inputs/empty.swift  2>&1 | %FileCheck %s
```

Comment:
You should either use `%target-build-swift` for these invocations or make sure that the job doesn't actually get executed (using `-###` or something), so that the test works correctly when only building a cross-compiler. (Which is something we do for iOS sometimes within Apple, but I don't remember if any of the public bots do it.)

Reply:
Added the -###.